openprescribing/templates/measure_for_all_entities.html

```diff
@@ -21,6 +21,11 @@ <h1>{{measure.name}} by all {{ entity_type_human }}s</h1>
     plus deciles across all {{ entity_type_human }}s in the
     NHS in England.
   </p>
+  <p>
+    <a href="{% url 'all_england' %}#{{ measure.id }}" class="btn btn-default">
+      View measure for NHS England combined &rarr;
```

Comment:
"combined" doesn't sound quite right to me.  Not quite sure what to suggest though.